pom.xml

```
@@ -21,6 +21,7 @@
   <properties>
     <surefire.forkCount>1</surefire.forkCount>
     <alphanetworkCompilerEnabled>false</alphanetworkCompilerEnabled>
+    <version.org.junit>5.4.2</version.org.junit>
```

@triceo I had to override version of junit as 5.5... is just silently skipping tests in this module .. at least those based on quarkus. Are you aware of that?

---

I am not. I will look into it. Thanks for pointing this out!